src/app/meetups/add-meetups/meetups-add.component.html

```diff
@@ -8,69 +8,67 @@
-      <div>
-        <mat-form-field>
+      <div class="form-width-3">
+        <mat-form-field class="full-width">
           <input matInput i18n-placeholder placeholder="Title" formControlName="title" required>
           <mat-error>
             <planet-form-error-messages [control]="meetupForm.controls.title"></planet-form-error-messages>
           </mat-error>
         </mat-form-field>
-      </div>
-      <div>
-        <mat-form-field>
+        <div>
+        <mat-form-field class="full-width">
           <textarea matInput i18n-placeholder placeholder="Description" formControlName="description" required></textarea>
           <mat-error>
             <planet-form-error-messages [control]="meetupForm.controls.description"></planet-form-error-messages>
           </mat-error>
         </mat-form-field>
       </div>
       <div>
-        <mat-form-field>
+        <mat-form-field class="half-width">
           <input matInput [matDatepicker]="datepickerStart" i18n-placeholder placeholder="Start Date" formControlName="startDate">
           <mat-datepicker-toggle matSuffix [for]="datepickerStart"></mat-datepicker-toggle>
           <mat-datepicker #datepickerStart></mat-datepicker>
           <mat-error><planet-form-error-messages [control]="meetupForm.controls.startDate"></planet-form-error-messages></mat-error>
         </mat-form-field>
-        <mat-form-field>
+        <mat-form-field class="half-width">
           <input matInput [matDatepicker]="datepickerEnd" i18n-placeholder placeholder="End Date" formControlName="endDate">
           <mat-datepicker-toggle matSuffix [for]="datepickerEnd"></mat-datepicker-toggle>
           <mat-datepicker #datepickerEnd></mat-datepicker>
           <mat-error><planet-form-error-messages [control]="meetupForm.controls.endDate"></planet-form-error-messages></mat-error>
         </mat-form-field>
       </div>
       <div>
-        <label i18n>Recurring</label>
+        <label i18n>Recurring Frequency  </label>
         <mat-radio-group formControlName="recurring">
-          <label i18n>Frequency</label>
-          <mat-radio-button value="daily" (change)="toggleDaily($event.value, false)" i18n>Daily</mat-radio-button>
-          <mat-radio-button value="weekly" (change)="toggleDaily($event.value, true)" i18n>Weekly</mat-radio-button>
-          <mat-radio-button value="none" (change)="toggleDaily($event.value, false)" i18n>None</mat-radio-button>
+          <mat-radio-button value="daily" (change)="toggleDaily($event.value, false)" il8n>Daily  </mat-radio-button>
+          <mat-radio-button value="weekly" (change)="toggleDaily($event.value, true)" i18n>Weekly  </mat-radio-button>
+          <mat-radio-button value="none" (change)="toggleDaily($event.value, false)" i18n>None  </mat-radio-button>
           <mat-error><planet-form-error-messages [control]="meetupForm.controls.recurring"></planet-form-error-messages></mat-error>
-        </mat-radio-group>
+        </mat-radio-group><br>
```

Comment:
This isn't a good use of the `<br>` tag (it should only be used for poetry & addresses), and the form looks good without this extra break.

In the future, use styles to add spacing between elements instead.